lib/fastlane/actions/changelog_from_git_commits.rb

```diff
@@ -56,6 +56,12 @@ def self.available_options
                                        optional: true,
                                        default_value: '%B',
                                        is_string: true),
+          FastlaneCore::ConfigItem.new(key: :match_pattern,
```

Comment:
Can we call this param `:tag_match_pattern` so that it is a little more clear what we're matching against?